samples/nrf9160/modem_shell/README.rst

```diff
@@ -538,6 +538,26 @@ Examples
 
 ----
 
+Running stored commands after startup
+=====================================
+
+MoSh command: ``startup_cmd``
+
+You can store up to 3 MoSh commands to be run after a start/bootup.
+As a default, commands are run after a default PDN context is getting active,
+but optionally can be set to be run N seconds after a bootup.
+
+Examples
+--------
+
+* An example of starting periodic location acquiring after a LTE has been connected with both cellular and gnss including sending location to cloud:
```

Comment:
```suggestion
* Starting periodic location acquiring after LTE has been connected with both cellular and GNSS including sending location to cloud:
```

Reply:
Edied to add a comma. 

Is this specifically to nRF Cloud or any cloud?